styles.css

```diff
@@ -902,14 +902,15 @@ form {
     margin-top: 12px;
     margin-bottom: 12px;
   }
+
   .know-text {
     list-style-type: none;
   }
 
   #languages {
     margin-top: 80px;
   }
-  
+
   .know-line {
     margin-top: 64px;
   }
```

Comment:
- [ ] The `desktop menu` is still visible on your mobile version. Kindly remove it as it is only supposed to appear on the desktop version of your portfolio

![image](https://user-images.githubusercontent.com/98453979/220945873-1f6d2a6a-2157-4fc2-b3ad-f355a72f4db3.png)

- [ ] The elements in the project cards on your `mobile version` appear to be overflowing the main container. Kindly fix that.

![image](https://user-images.githubusercontent.com/98453979/220946513-bb5f7f19-46b3-4cd5-a767-866ba666dc04.png)